internal/config/cgroupmanager/systemdmanager.go

```diff
@@ -0,0 +1,148 @@
+// +build linux
```

Comment:
Maybe file name should `systemd_manager.go` (or just `systemd.go`)?

The directory and the package can just  be `cgroup` or `cgroupmgr` or `cgmgr` as per recommendation from https://blog.golang.org/package-names

Reply:
I went for cgmgr for the package and systemd/cgroupfs for the file names